__tests__/components/signup.test.js

```diff
@@ -34,6 +42,9 @@ describe('Signup Components tests...', () => {
     expect(signup.state()).toBeDefined();
     mapStateToProps({});
   });
+  it('Should test validation', () => {
```

Comment:
@rukundoeric Specifically, what validations are tested here? Additionally, this test does not have assertions.